snap/snapcraft.yaml

```diff
@@ -117,59 +117,71 @@ parts:
     prime:
       - -lib/systemd/system/*
 
+  flutter-git:
+    source: https://github.com/flutter/flutter.git
+    source-branch: stable
+    source-depth: 1
+    plugin: nil
+    override-build: |
+      mkdir -p $SNAPCRAFT_PART_INSTALL/usr/bin
+      mkdir -p $SNAPCRAFT_PART_INSTALL/usr/libexec
+      cp -r $SNAPCRAFT_PART_SRC $SNAPCRAFT_PART_INSTALL/usr/libexec/flutter
+      ln -s $SNAPCRAFT_PART_INSTALL/usr/libexec/flutter/bin/flutter $SNAPCRAFT_PART_INSTALL/usr/bin/flutter
+      flutter config --enable-linux-desktop
+      flutter doctor -v
+    build-packages:
+      - clang
+      - cmake
+      - curl
+      - libgtk-3-dev
+      - ninja-build
+      - unzip
+      - xz-utils
+      - zip
+    stage-packages:
+      - clang
+      - cmake
+      - curl
+      - libgtk-3-dev
+      - ninja-build
+      - unzip
+      - xz-utils
+      - zip
+    override-prime: ''
```

Comment:
This is only for building so we don't want to prime anything. I tried `- -"*"` but it errors out because of some flutter binary cache files. I saw this in Olivier's `firefox-snap-experiments` and it seems to do the trick.

Reply:
You should drop all of those stage-packages in the flutter-git part.